README.md

```diff
@@ -296,3 +296,15 @@ bundle exec rspec spec/functional
 ## Wrong ruby version
 
 Ruby 1.8.7 or higher is required.
+
+## Warning message: To disable read timeouts, please set timeout to nil instead of -1
+
+This message is generated by the rest-client gem if the timeout is not set to nil.
+The version of the gem that is compatible with ruby 1.8 and that which is compatible with
+ruby 1.9 and greater behave differently.  The previous revision requires a -1, while the new
+version requres nil.  You can silence the warning message by passing
+```
+:timeout => nil
+```
+when creating the right_api_client.
+
```

Comment:
So, the real problem here (i.e. the reason we cannot set the DEFAULT_TIMEOUT to nil) is because of the following:

Under Ruby 1.8, rest-client v1.6.x is used which has the following behavior for timeout values:
- nil: 60s (defaults to behavior of Net::HTTP)
- -1: infinite

Under Ruby 1.9, rest-client v1.7.x may be used which has the following behavior for timeout values:
- nil: infinite
- -1: infinite (+ results in the annoying warning being printed)

So we should tell the RAC users to set the `:timeout => nil` ONLY if they are using rest-client v1.7 or higher. For rest-client 1.6.x (highest version supporting Ruby 1.8), that will result in a 60s timeout instead of infinite due to rest-client behavior at that time.

Hope this clarifies!